python_modules/dagster/dagster/_core/storage/runs/sql_run_storage.py

```diff
@@ -955,6 +955,27 @@ def get_backfills(
             )
         return backfill_candidates
 
+    def get_backfills_count(self, filters: Optional[BulkActionsFilter] = None) -> int:
+        check.opt_inst_param(filters, "filters", BulkActionsFilter)
+        if filters and filters.tags:
+            # runs can have more tags than the backfill that launched them. Since we filtered tags by
+            # querying for runs with those tags, we need to do an additional check that the backfills
+            # also have the requested tags. This requires fetching the backfills from the db and filtering them
+            query = self._backfills_query(filters=filters)
```

Comment:
this is a bit of a bummer that i'm fetching the actual backfill objects from the db, but i'm not sure how else to check that the tags are actually on the backfill object and not just the runs that are launched

Reply:
Can you remind me: how do backfill runs have more tags than the backfills themselves?

Reply:
if you have something like 
```python 
@asset(tags={"foo": "bar"})
...
```
 then a for a backfill of the asset, all of the runs will have the `foo:bar` tag but the backfill object itself won't have that tag